projects/fsastorefrontlib/src/shared/util/helpers/pipe/parseDate.pipe.ts

```diff
@@ -3,12 +3,14 @@ import { Pipe, PipeTransform } from '@angular/core';
 @Pipe({ name: 'parseDate' })
 export class ParseDatePipe implements PipeTransform {
   transform(input: string): Date {
-    const trimmedDate = input.split(' ');
-    if (trimmedDate.length === 6) {
-      trimmedDate.splice(4, 1);
-      return new Date(trimmedDate.join(' '));
-    } else {
-      return new Date(input);
+    if (input) {
+      const trimmedDate = input.split(' ');
+      if (trimmedDate.length === 6) {
+        trimmedDate.splice(4, 1);
+        return new Date(trimmedDate.join(' '));
+      } else {
```

Comment:
Doesn't need `else` here. If it doesn't enter in first if, it is enough just to put `return new Date(input)` without else.